magento/magento2
# [MFTF] Refactoring AdminMassOrdersUpdateCancelPendingOrderTest

### Description (*)
The test is refactored to improve execution time

### Related Pull Requests
https://github.com/magento/magento2/pull/31509

### Manual testing scenarios (*)
1. Create an order
2. Go to the Orders Grid
3. Try to cancel the order via mass update
4. Verify the result

### Resolved issues:
1. [x] resolves magento/magento2#31514: [MFTF] Refactoring AdminMassOrdersUpdateCancelPendingOrderTest
